src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/Interop.Libraries.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace Internal.Cryptography.Pal.Native
+{
+    internal static partial class Interop
+    {
+        private static class Libraries
```

Comment:
Why do we have this Libraries rather than using the main one in Common\src\Interop\Windows?